lib/screens/config/v2/evergreen_content_item.ex

```diff
@@ -1,18 +1,23 @@
 defmodule Screens.Config.V2.EvergreenContentItem do
   @moduledoc false
 
+  alias Screens.Config.V2.Schedule
   alias Screens.V2.WidgetInstance
 
   @type t :: %__MODULE__{
           slot_names: list(WidgetInstance.slot_id()),
           asset_path: String.t(),
-          priority: WidgetInstance.priority()
+          priority: WidgetInstance.priority(),
+          schedule: list(Schedule.t())
         }
 
   @enforce_keys ~w[slot_names asset_path priority]a
-  defstruct @enforce_keys
+  defstruct slot_names: [],
+            asset_path: "",
+            priority: nil,
```

Comment:
The values here are fine but won't actually be used since their keys must be supplied when creating this struct—if you want you can just make all 3 of them `nil`.